sympy/core/tests/test_arit.py

```diff
@@ -1465,10 +1465,12 @@ def test_Pow_is_nonpositive_nonnegative():
 
     assert (x**2).is_nonnegative is True
     i = symbols('i', imaginary=True)
-    assert (i**2).is_nonpositive is True
-    assert (i**4).is_nonpositive is False
+    assert (i**2).is_nonpositive is None
+    assert (i**4).is_nonpositive is None
+    # XXX: Should this be None?
     assert (i**3).is_nonpositive is False
```

Comment:
This is the only example I'm aware of where we have a wrong result from the assumptions system itself. If `i = x*I` for real `x` then `i**3` is `-I*x**3` which could be zero if `x` is zero. Since zero is nonpositive this should give None.

Reply:
```python
In [21]: i = symbols('i', imaginary=True)

In [22]: (i**3)._eval_is_extended_real()
Out[22]: False
```